rules/aircraft.yaml

```diff
@@ -121,32 +121,41 @@ zep:
 	AnnounceOnBuild:
 	Voiced:
 		VoiceSet: KirovVoice
-#	SpawnActorOnDeath:
-#		Actor: zephusk
+	SpawnActorOnDeath:
+		Actor: zephusk
 	Targetable@ground:
 		TargetTypes: Ground
 		UpgradeTypes: airborne
 		UpgradeMaxEnabledLevel: 0
 
-#zephusk:
-#	Inherits: ^HelicopterHusk
-#	Tooltip:
-#		Name: Kirov Airship
-#	RenderSprites:
-#		Image: zep
-#	RenderVoxels:
-#		Image: zep
-#	WithVoxelBody:
-#	WithShadow:
-#	Helicopter:
-#		ROT: 10
-#		Speed: 50
-#		AirborneUpgrades: airborne
-#	RevealsShroud:
-#		Range: 8c0
-#		Type: CenterPosition
-#	FallsToEarth:
-#		Velocity: 200
+zephusk:
+	Tooltip:
+		Name: Kirov Airship
+	Armor:
+		Type: Light
+	Health:
+		HP: 140
+	Aircraft:
+		TurnSpeed: 10
+		Speed: 50
+		AirborneUpgrades: airborne
+		CanHover: True
+		Voice: Move
+	FallsToEarth:
+		Spins: False
```

Comment:
IIRC that is not true, but I had issues to set this up and this probably needs an upstream fix.
(The kirov begins to spin faster and faster which looks very weird.)
Just speaking my "thoughts" out loud.